pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -172,7 +172,6 @@
     }
 
     protected List<String> internalGetPartitionedTopicList() {
-        validateAdminAccessForTenant(namespaceName.getTenant());
```

Comment:
If we are going to remove this line, we should update the annotations on the two methods that call `internalGetPartitionedTopicList()`.

Reply:
Nice!
In addition, we should also update the annotations on the method that call `internalGetList()`.
https://github.com/apache/pulsar/blob/01d73aa90a4dd13f6b4306ef8e52e1dce7ee401d/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java#L149-L150

Reply:
Good catch!

Reply:
I have updated all the annotations, PTAL again, thx!